packages/gluestick/src/config/compileWebpackConfig.js

```diff
@@ -22,9 +22,6 @@ const readRuntimePlugins = require('../plugins/readRuntimePlugins');
 const readServerPlugins = require('../plugins/readServerPlugins');
 const callHook = require('../renderer/helpers/callHook');
 const { requireModule } = require('../utils');
-const {
```

Comment:
universal-webpack uses an incompatible way of referencing node externals, and doesn't do a lot for us, anyway. Note the large block of code below which is getting around its current issues.